agreement_legal/static/src/xml/domain_widget_view.xml

```diff
@@ -0,0 +1,307 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template" xml:space="preserve">
+    <t t-name="FieldDomain.content">
+        <t t-if="partialUse">
+            <div t-if="hasModel" class="o_field_domain_panel">
```

Comment:
Should we remove this comment section?

Reply:
hello, shouldn't "agreement_legal/data/demo.xml" only be loaded if demo data is set?